### PR TITLE
EWI Viral Load Supression Report

### PR DESCRIPTION
**Pull request by balukujr**
https://trello.com/c/BLSaL6p6

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupPillPickupIndicator.java [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+import org.springframework.stereotype.Component;
+
+import java.util.*;
+
+import static org.openmrs.module.ugandaemrreports.reporting.metadata.Dictionary.getConcept;
+
+/**
+ */
+@Component
+public class SetupPillPickupIndicator extends UgandaEMRDataExportManager {
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupPillPickupIndicator.java]: @balukujr Rename to SetupEWIPillPickupReport not indicator

File: api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupViralLoadSupression.java [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+import org.openmrs.module.ugandaemrreports.reporting.metadata.Dictionary;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.*;
+
+/**
+ */
+@Component
+public class SetupViralLoadSupression extends UgandaEMRDataExportManager {
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupViralLoadSupression.java]: @balukujr not needed in this PR

File: omod/src/main/resources/apps/ugandaemr_reports_app.json [hunk 3]
```diff
@@ -79,7 +79,16 @@
         "label": "Appointments List",
         "url": "reportingui/runReport.page?reportDefinition=9c85e20b-c3ce-4dc1-b332-13f1d02f1c5c",
         "order": 5
-      },{
+      },
```
Comment (ssmusoke) [inline omod/src/main/resources/apps/ugandaemr_reports_app.json]: @balukujr Remove all Viral Load reports not needed in this PR

**Comment by balukujr** [comment]
Resolved all issues @ssmusoke  noted above

**Pull request merged**